apollo/typeDefs/form.ts

```diff
@@ -4,7 +4,40 @@ export const form = gql`
   type Form {
     id: ID!
     name: String!
+    questions: [Question]!
   }
+
+  type QuestionData {
+    question: String!
+    required: Boolean!
+    type: QuestionType!
+  }
+
+  type Question {
+    id: ID!
+    type: QuestionType!
+    data: QuestionData!
+    createdAt: String!
+  }
+
+  enum QuestionType {
+    text
+  }
+
+  input QuestionCreateParams {
+    question: String!
+    required: Boolean!
+    type: QuestionType!
+  }
+
+  extend type Mutation {
+    createQuestion(
+      workspaceId: String!
+      formId: String!
+      question: QuestionCreateParams!
+    ): Workspace!
+  }
+
```

Comment:
I would recommend we group all the mutations on line 42 since we're already extending the mutation type for the form schema.


```
 extend type Mutation {
    createForm(workspaceId: String!): Form!
    createQuestion(workspaceId: String!, formId: String!, question: QuestionCreateParams!): Workspace!
  }
```